src/widgets/listview/GenericListView.cpp

```diff
@@ -21,6 +21,8 @@ GenericListView::GenericListView()
 
             this->requestClose();
         });
+
+    this->refreshTheme(*getTheme());
```

Comment:
Don't call this in the ctor, instead let the inputcompletionpopup call its themechangedevent in its constructor like the quick switcher does